src/rgw/rgw_main.cc

```diff
@@ -345,6 +345,8 @@ int main(int argc, const char **argv)
 			   set_logging(new RGWRESTMgr_SWIFT));
   }
 
+  rest.register_resource("info", set_logging(new RGWRESTMgr_SWIFT_Info));
+
```

Comment:
@pritha-srivastava: take a look on a [similar code](https://github.com/ceph/ceph/pull/10069/files#diff-703174d51f9d2cd706c9e59b8e7ce8f0R344) in the `crossdomain.xml` implementation.